course/lesson-11-time-delta/rotating-using-delta/TestsRotatingDelta.gd

```diff
@@ -22,20 +22,89 @@ func _prepare() -> void:
 
 func test_rotating_character_is_time_dependent() -> String:
 	if not _has_proper_body:
+		#reverse find for any delta at all
 		var has_delta: bool = _body.rfind("delta") > 0
```

Comment:
You can use type inference in cases like these:

```gd
var has_delta := _body.rfind("delta") > 0
```
We recommend that because the bool type is implied by the expression, and it'll be registered by Godot which will be able to document this variable as being a bool